luigi/file.py

```diff
@@ -81,14 +81,16 @@ def __init__(self, path=None, format=None, is_tmp=False):
         self.format = format
         self.is_tmp = is_tmp
 
+    def mkdirs(self):
```

Comment:
I think mkdirs is enough and it is on the same level of understanding as open,close,copy,move. Surely could rename to `makedirs` mimicking the actual os call `os.makedirs` if it helps.